multiversx_sdk/relayed/relayed_controller.py

```diff
@@ -26,6 +26,7 @@ def create_relayed_v1_transaction(
         transaction.nonce = nonce
 
         self._set_version_and_options_for_hash_signing(sender, transaction)
+        self._add_extra_gas_limit_if_required(transaction)
```

Comment:
Oh, yes, since this can also be guarded.

Can we also add deprecation warnings? Can also be in a separate PR.

Reply:
Will do in a separate PR.